janus/lib/commands.rb

```diff
@@ -15,12 +15,14 @@ def setup(config)
   end
 
   class AddUser < Etna::Command
-    usage '<email> <first_name> <last_name> [<password>]'
-    def execute(email, first_name, last_name, password=nil)
-      user = User.find_or_create(email: email)
+    usage '<email> <name> [<password>]'
```

Comment:
We will also want to update the `seed_janus` script to ensure it only passes in a quoted full name as opposed to separate first and last name arguments.